lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -69,6 +69,11 @@ def scrub(transcript)
           gsub(%r((\"cvv\\\":\\\")\d+), '\1[FILTERED]')
       end
 
+      def inquiry_request(parameters, options)
```

Comment:
I would suggest changing this method name to `inquire`, to follow the pattern that these public methods are named for a verb/action. We will use the same method name across all other gateways we add this into.

The [worldpay code](https://github.com/activemerchant/active_merchant/blob/e50f4cd30acd60c54f9c5783f2a06e668634e246/lib/active_merchant/billing/gateways/worldpay.rb#L178) this is modeled after is a private method that is called by worldpay's action methods, so worldpay will also need the addition of this public method eventually.

Reply:
Will change that, I thought I wrote `inquire`

Reply:
Just to clarify, I would also remove `_request`, so the method is just called `inquire`. This better matches authorize, purchase, capture, void, etc.